code/src/pages/products/ProductsPage.tsx

```diff
@@ -41,29 +46,28 @@ const ProductsPage = () => {
     setSearchParams({ sort: value });
   };
 
-  const pagesCount = productsResponse?.totalPages ?? 1;
-  const productsCount = productsResponse?.totalElements ?? 0;
-
-  const paginationBlock = pagesCount > 1 && (
-    <AppContainer className="spa-products-page__pagination">
-      <AppPagination page={page} count={pagesCount} size="large" />
-    </AppContainer>
-  );
+  const productsCount = filteredProductsList?.length ?? 0;
 
   return (
     <PageWrapper>
       <AppBox className="spa-products-page" data-cy="products-page">
         <AppTypography
           variant="h3"
           className="spa-products-page__header"
-          translationKey="productsAll.label"
+          translationKey={
+            !categoryType ? "productsAll.label" : `productsAll.${categoryType}`
```

Comment:
Usually we move ternary conditions like this in variable above return statement. It makes code more readable.
Something like this before return statement ```const productsAllLabel = !categoryType ? "productsAll.label" : `productsAll.${categoryType}```
And then in jsx `translationKey={productsAllLabel}`

Reply:
I moved the ternary operators into variables, in order to improve readability.